samples/snippets/standard/data/sqlite/DeferredTransactionSample/DeferredTransactionSample.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="5.0.0-rc.1.20451.13" />
```

Comment:
Should I add a `NuGet.config` file to reference the daily builds feed or wait for RC1 to be published to nuget.org?

Reply:
We just updated our CI build to RC1, so this is ready to :shipit: